### PR TITLE
/pathways page

### LINKED ISSUE #247: Build Pathways page
My suggestion:
 * Use the same design as the Spaces page, except Pathways won't have icons. 
 * Replace "Wranglers" with "Pathfinders," but @Saallen will work to collect Pathfinder bios, so we can use the same design pattern there as well.
 * Will need a different header image.
 * url: /pathways
 * Link to this page from the /sessions page now described in #176

cc @mmmavis @Saallen @cassiemc

### PR DESCRIPTION
**Pull request by mmmavis**
fixes #247

### CODE REVIEW

File: less/pages/sessions.less [hunk 1]
```diff
@@ -4,61 +4,3 @@
     margin-bottom: 0;
   }
 }
-
```
Comment (mmmavis) [inline less/pages/sessions.less]: moved this chunk to `shared.less`

File: components/space-pathway-profile.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          { this.props.type ? <div className="type">{this.props.type}</div> : null }
+          <div className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></div>
+        </div>
+        <div className="contacts">
+          <h2>{(this.props.contacts && this.props.contacts.length > 1) ? this.props.contactTitle + "s" : this.props.contactTitle}</h2>
+          <ul>
+          {
+            this.props.contacts ? this.props.contacts.map(function(contact,i) {
+              return (
+                <li key={i}>
```
Comment (Pomax) [inline components/space-pathway-profile.jsx]: never do this. The key is for React to do fast dom diffs, so the index in the array means nothing, the key has to be unique to the thing you're generating. So, `contact.name` for instance.

File: components/space-pathway-profile.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+var SpacePathwayProfile = React.createClass({
+  render: function() {
+    return (
+      <div className="space-pathway-profile">
+        <div className="detail">
+          <div className="header">
+            <h1>{this.props.name}</h1>
+          </div>
+          { this.props.type ? <div className="type">{this.props.type}</div> : null }
+          <div className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></div>
```
Comment (Pomax) [inline components/space-pathway-profile.jsx]: does `description` contain HTML code? If not, let's not use this. If it does: that feels dangerous, why does it contain raw HTML?

File: pages/pathways.jsx [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+var Footer = require('../components/footer.jsx');
+var HeroUnit = require('../components/hero-unit.jsx');
+var SpacePathwayProfile = require('../components/space-pathway-profile.jsx');
+
+var PathwaysInfo = [
+  {
+    name: "Online Learning Done Face-to-Face",
+    type: "Independent Pathway",
+    description: 
+      "<p>Online learning offers tremendous opportunity, but often lacks a key component — the ability to collaborate with those around us. In this Pathway, we explore “Learning Circles,” or in-person study groups that complement online learning.</p>",
```
Comment (Pomax) [inline pages/pathways.jsx]: We probably don't want to encode raw HTML, but instead use:
```
description: [
  "Online learning .... "
],
contacts: [
...
],
```
and then in our JSX we can generate:
```
<div>
{ description.map(function(text) { return <p>{text}</p>; }) }
</div>
```

File: pages/pathways.jsx [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+];
+
+
+var PathwaysPage = React.createClass({
+  pathways: PathwaysInfo,
+  componentWillMount: function() {
+    this.pathways = this.pathways.map(function(pathway,i) {
+      var whiteBg = (i%2==0) ? "white-background" : "";
+      return (
+        <div className={whiteBg} key={i}>
```
Comment (Pomax) [inline pages/pathways.jsx]: don't use `key={i}`, here it'd be something like `key={pathway.name}` or something

File: components/space-pathway-profile.jsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  render: function() {
+    return (
+      <div className="space-pathway-profile">
+        <div className="detail">
+          <div className="header">
+            <h1>{this.props.name}</h1>
+          </div>
+          { this.props.type ? <div className="type">{this.props.type}</div> : null }
+          <div className="description">
+            { this.props.description.map(function(text) { return <p>{text}</p>; }) }
```
Comment (Pomax) [inline components/space-pathway-profile.jsx]: this looks to be missing a `key`

**Comment by Pomax** [comment]
I think.. this is R+!

but probably best to squash before merging

**Comment by mmmavis** [comment]
SQUASHED

**Comment by mmmavis** [comment]
:tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada:

**Pull request merged**